content/account/dashboard/notifications.textile

```diff
@@ -0,0 +1,96 @@
+
+* Go to the "Apple Developer Program.":https://developer.apple.com/programs/
+* Use the Apple developer portal to create a *push notification* service certificate for your app.
+* Export the certificate as a *.p12 file*.
+* Next, you can either import the *.p12 cert* or create a *PEM file* and copy it into your Ably dashboard:
+
+* Import the *.p12 file*:
+** In your Ably "dashboard":https://ably.com/accounts, navigate to the *Notifications* tab under your app settings.
+** Go to *push notifications setup*, click *configure push* and scroll to the *setting up Apple push notification service* section.
+** Select the *.p12 file* you exported and enter the password you created during the export process.
+** Click *save*. You should receive confirmation that the certificate has been successfully imported.
+** To further confirm the import, refresh the page and check if the *PEM cert* and *private key* text boxes are now populated with the imported key details.
+* Create a *PEM file* from the *.p12 file*:
+** Using "OpenSSL":/https://www.openssl.org/, convert the recently exported *.p12 file* (@io.ably.test.p12@) to a *PEM file* with the following command: @$ openssl pkcs12 -in ./io.ably.test.p12 -out ./io.ably.test.pem -nodes -clcerts@.
+** Open the *PEM file* in your text editor.
+** Copy everything between and including @-----BEGIN CERTIFICATE-----@ and @-----END CERTIFICATE-----@, and paste it into the *PEM cert* text box of the Apple push notification service section of your Ably notifications app "dashboard":https://ably.com/accounts.
+** Similarly, copy everything between and including @-----BEGIN PRIVATE KEY-----@ and @-----END PRIVATE KEY-----@, and paste it into the *PEM private key* text box of the same section. Then, click *save*.
+
+h3(#configure-web). Configure Web Push for web browsers
+
+Configuration is the first step in setting up push notifications with Ably. This step requires setting up the necessary infrastructure for your application.
+
+The following steps configure a web browser for push notifications using the Ably JavaScript SDK:
+
+* Generate a VAPID (Voluntary Application Server Identification) Key:
+** When you first activate a web browser using the Ably JavaScript "SDK":https://ably.com/docs/getting-started/setup, Ably automatically creates a VAPID key pair for your application.
+** Ably will then use the same VAPID key pair for all subsequent activations in the same application.
+* Service Worker Setup:
+** You must host a "service worker":https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API to use to push messages.
+** Service workers operate in the background, outside of any specific page or tab context.
+** The service worker will handle all incoming push notifications as JavaScript events.
+* Implement the service worker:
+** Create a JavaScript file to act as your service worker.
+** In your service worker file, listen for push events using @addEventListener("push", callback)@.
```

Comment:
Nothing is configured in the dashboard for Web Push I don't think, is it?

Reply:
True!